ocs_ci/framework/conf/default_config.yaml

```diff
@@ -104,6 +104,11 @@ ENV_DATA:
   # measurement_dir: '/tmp/ocs_ci_monitoring_measurement/'
   # Default RHCOS image to be used for VmWare deployment
   vm_template: 'rhcos-4.3.0-x86_64-vmware'
+  # minimal write speed of fio as used in workload_fio_storageutilization
+  # fixtures, which is used to compute timeout of the write job (so that
+  # when actual write speed of the operation is smaller than this value, the
+  # workload fixture will fail on a timeout)
+  fio_storageutilization_min_mbps: 30.0
```

Comment:
This speed is assumed for all aws configurations, right? Maybe it would be good to mention it in a comment.

Reply:
Yes, will add this note.

Reply:
done in 8a3ad0fb0437938107ef9b8cc192f545bdfaa15b